src/content/en/2020/performance.md

```diff
@@ -92,30 +98,32 @@ In the chart above, we can observe that between 43% and 53% of websites have goo
   caption="Aggregate LCP performance split by country.",
   description="Bar chart showing that the highest percentages of good LCP performance are distributed between European and Asian countries. The Republic of Korea leads with 76% good readings, while India is last with 16% of good readings.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1936626175&format=interactive",
-  sheets_gid="2077755325",
+  sheets_gid="263037691",
   sql_file="web_vitals_by_country.sql"
   )
 }}
 
-The **highest percentage of good LCP readings is mostly distributed amongst European and Asian countries** with the Republic of Korea (South Korea) leading at 76% of good metric readings. South Korea is a consistent leader in mobile speeds, with an impressive 145 Mbps download [reported by Speedtest Global Index](https://www.speedtest.net/global-index) for October. Japan, Czechia, Taiwan, Germany and Belgium are also a handful of countries with reliably high mobile speeds. Australia, while leading in mobile network speeds, is let down by slow desktop connections and latency which places it at the bottom section of the ranking above.
+{# TODO - why not the full list? #}
 
-India remains the last one in the set with only 16% of good experiences. While the population of people connecting to the internet for the first time is continually growing, the mobile and desktop network speeds are [still an issue](https://www.opensignal.com/reports/2020/04/india/mobile-network-experience), with average downloads of 10Mbps for 4G, 3Mbps for 3G and below 50Mbps for desktop.
+The highest percentage of good LCP readings is mostly distributed amongst European and Asian countries with the Republic of Korea (South Korea) leading at 76% of good metric readings. South Korea is a consistent leader in mobile speeds, with an impressive 145 Mbps download [reported by Speedtest Global Index](https://www.speedtest.net/global-index) for October. Japan, Czechia, Taiwan, Germany and Belgium are also a handful of countries with reliably high mobile speeds. Australia, while leading in mobile network speeds, is let down by slow desktop connections and latency which places it at the bottom section of the ranking above. {# TODO should we split this by desktop and mobile to prove this? Or at least state that we did look at this to confirm this? #}
```

Comment:
@thefoxis thoughts?

Reply:
@bazzadp I don't think charting this is necessary. elsewhere, we don’t have charts to prove the anecdotal per-country advice. at this point it is just an anecdote :) but you can see it in the link preceding this sentence, where AU is 5th on mobile and 60th on desktop in terms of speeds

Reply:
No probs. Resolving.